pkg/arvo/tests/sys/zuse/html.hoon

```diff
@@ -82,11 +85,754 @@
         ==
       ==
     !>  "<html><head><title>My first webpage</title></head><body><h1>Welcome!</h1>Hello, world!\0aWe are on the web.\0a<div></div><script src=\"http://unsafely.tracking.you/cookiemonster.js\"></script></body></html>"
-    :: Attributes
+    ::  Attributes
     ::
     %+  expect-eq
       !>  "<input type=\"submit\">Submit</input>"
       !>  %-  en-xml:html
       ;input(type "submit"): Submit
   ==
+::  JSON encoding/decoding
+::
+++  from-code-points
+  |=  points=(list @)
+    (tufa `(list @c)`points)
+++  from-code-point
+  |=  point=@
+    (tuft point)
++$  json-parse-spec  [name=tape input=cord expected=json]
++$  json-parse-rejection-spec  [input=cord name=tape]
+::  For checking a large list of examples against expected values.
+::  It also nicely formats any failures.
+::
+++  run-parse-specs
+  ::  legend tells of a man who made the Kessel run in less than 12
+  ::  parse-specs...
+  |=  specs=(list json-parse-spec)
+  %-  zing
+  %-  turn
+  :-  specs
+  |=  spec=json-parse-spec
+  ^-  tang
+  =+  result=(expect-eq !>(`expected.spec) !>((de-json:html input.spec)))
+  ?~  result  ~
+  `tang`[[%leaf "in {name.spec}:"] result]
+++  run-parse-rejection-specs
```

Comment:
Comments aren't necessary for every arm (IMO), but they should at least be set off by blank comment lines.